fluentui_peoplepicker/src/main/java/com/microsoft/fluentui/peoplepicker/PeoplePickerTextView.kt

```diff
@@ -612,18 +632,20 @@ internal class PeoplePickerTextView : TokenCompleteTextView<IPersona> {
 
     private class TokenListener(val view: PeoplePickerTextView) : TokenCompleteTextView.TokenListener<IPersona> {
         override fun onTokenAdded(token: IPersona) {
-            if (view.shouldAnnouncePersonaAddition)
+            if (view.shouldAnnouncePersonaAdditionMap[token] != false)
```

Comment:
Where are we setting this to true?

Reply:
We don't need to, as announcements are ON by default.